tree/dataframe/inc/ROOT/RDF/RLoopManager.hxx

```diff
@@ -113,7 +113,15 @@ using ROOT::RDF::RDataSource;
 /// This class is responsible of running the event loop.
 class RLoopManager : public RNodeBase {
    using ColumnNames_t = std::vector<std::string>;
-   enum class ELoopType { kROOTFiles, kROOTFilesMT, kNoFiles, kNoFilesMT, kDataSource, kDataSourceMT };
+   enum class ELoopType {
+      kNoData,
```

Comment:
The difference between `kNoData` and `kNoFiles` might become ambiguous/confusing. What about `fInvalid` or `fNotRunnable` or something explicit like this to clearly signal the implication of this loop type?